SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -48,6 +49,7 @@ Provides:       /usr/sbin/nologin
 Provides:       /usr/sbin/useradd
 Provides:       /usr/sbin/userdel
 Provides:       passwd = %{version}-%{release}
+Conflicts:      util-linux < 2.37.4-3
```

Comment:
Nit: It would be nice to have a comment here explaining the conflict.

Reply:
I've added a comment to the conflict in util-linux.spec, and removed this conflict, since there would be no file conflicts with the new shadow-utils and old util-linux